controller/namedusers.go

```diff
@@ -94,11 +94,17 @@ func (c *NamedusersController) Deactivate(ctx *app.DeactivateNamedusersContext)
 			"err":      err,
 			"username": ctx.Username,
 		}, "error occurred while deactivating user")
-		return jsonapi.JSONErrorResponse(ctx, err)
+		if notFound, _ := errors.IsNotFoundError(err); !notFound {
+			return jsonapi.JSONErrorResponse(ctx, err)
+		}
+		// Ignore notFound error. If user has not found then we don't want the reg-app keep trying to deactivate it in auth.
+		// We can add some dummy Identity in response here
+		identity = &repository.Identity{}
```

Comment:
Don't you also need to set the `User` property, since that is what is returned in line 109?

Reply:
User will be empty (not nil). Reg app doesn't use the returned identity/user so we can trick our API by returning an empty identity/user.